packages/documentation/src/elements/tag/tag.stories.ts

```diff
@@ -0,0 +1,30 @@
+import type { Meta, StoryObj } from "@storybook/web-components";
+import { html } from "lit";
+
+interface TagArgs {
+  label: string;
+}
+
+const meta: Meta<TagArgs> = {
+  args: {
+    label: "Tag Label",
+  },
+};
+
+export default meta;
+type Story = StoryObj<TagArgs>;
+
+export const Default: Story = {
+  render: (args) => html`<span class="hap-tag">${args.label}</span>`,
+};
+
+export const LeadingIcon: Story = {
+  render: (args) => html`
+    <span class="hap-tag">
+      <svg viewBox="0 0 24 24" class="hap-icon hap-icon--small">
```

Comment:
Hier sollte `renderIcon('sparkles')` verwendet werden

Reply:
Guter Punkt. Hatte die Funktion gar nicht auf dem Schirm... Das intended Typing für `renderIcon` hat nicht funktioniert. Obwohl der Typ aus der verfügbaren indexiert hat, war der Typ einfach nur ein String. Ist in [13b6b9c](https://github.com/holisticon/atomic-playfulness/pull/134/commits/13b6b9c9ef58775cde200cb753d1b3f2f8469f0b) mit gefixed.